src/Behat/Behat/Snippet/Cli/SnippetsController.php

```diff
@@ -139,6 +143,14 @@ public function printAllSnippets()
         $this->writer->printSnippets($this->printer, $snippets);
     }
 
+    private function printSnippetGenerationFailures(): void
+    {
+        $failures = $this->registry->getGenerationFailures();
+        count($failures) && $this->output->writeln('');
+
+        $this->printer->printSnippetGenerationFailures($failures);
```

Comment:
Although there is a `SnippetPrinter` interface, which does not contain this new method, the `SnippetsController` depends directly on `ConsoleSnippetPrinter` so this is BC without any additional checks required.